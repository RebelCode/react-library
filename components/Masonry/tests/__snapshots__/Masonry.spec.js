import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Masonry } from "../../index";

afterEach(cleanup);

describe("Skeleton", () => {
  const Comp = (
    <Masonry columns={2} gap={2} aria-label="Masonry">
      <h1>One</h1>
      <h1>Two</h1>
    </Masonry>
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("Masonry")).toBeTruthy();
  });
});
