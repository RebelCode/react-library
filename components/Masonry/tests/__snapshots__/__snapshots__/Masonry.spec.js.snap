// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should take a snapshot 1`] = `
<DocumentFragment>
  <div
    aria-label="Masonry"
    class="root"
    style="grid-template-columns: repeat(2, minmax(0, 1fr)); column-gap: 2px;"
  >
    <div
      class="column"
    >
      <div
        class="cell"
        style="margin-bottom: 2px;"
      >
        <h1>
          One
        </h1>
      </div>
    </div>
    <div
      class="column"
    >
      <div
        class="cell"
        style="margin-bottom: 2px;"
      >
        <h1>
          Two
        </h1>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
